apps/privacy-panel/templates/rp/change_pass.html

```diff
@@ -13,8 +13,12 @@ <h1 data-l10n-id="rp-panel"></h1>
         </li>
         <li>
           <form id="rp-change-pass-form">
-            <select class="pin-type"></select>
-            <input class="pin" x-inputmode="digit" size="8" maxlength="8" type="password" placeholder="">
+          <li>
+            <span class="button icon icon-dialog">
+              <select class="pin-type"></select>
+            </span>
+          </li>            
+            <input class="pin" x-inputmode="digit" size="8" maxlength="8" type="password" data-l10n-id="enter" placeholder="Enter Code">
```

Comment:
Please remove the placeholder attribute content. I think we should just rely on l10n to locaize it - that way we will also make sure that it's truly getting localized.